src/pages/Delivery/index.jsx

```diff
@@ -36,50 +37,50 @@ const Delivery = () => {
     destination_confirmed: { headerTitle: 'Destino alcanzado', headerSubtitle: 'Hemos llegado' },
   }
 
-  if (loadingPack) {
-    return <LoadingView />
-  }
-
-  if (errorPack) {
-    return <Modal
-      isOpen={true}
-      handleClick={() => { history.push('/check') }}
-      actionText="Regresar"
-    >
-      <h1>Este paquete no existe</h1>
-    </Modal>
-  }
-
   return (
-    <Layout
-      packageId={id}
-      headerTitle={headerStatus[currentStatus]?.headerTitle}
-      headerSubtitle={headerStatus[currentStatus]?.headerSubtitle}
-      clientAddress={packageInformation?.packages[0]?.client_address}
-      estimatedArrival={packageInformation?.packages[0]?.estimated_arrival}
-      isLoading={isLoading}
-      toChat={toChat}
-      DeliveryConfirmedModal={
+    <Fragment>
```

Comment:
Why you use `<Fragment>` instead just `<>`?

Reply:
Yeah, no problem. I like that